File: src/main/java/pl/training/workshop/m4/s06_inlinemethod/step2/OnlineTicketPricing.java
package pl.training.workshop.m4.s06_inlinemethod.step2;

import java.math.BigDecimal;

/** Sprzedaż internetowa: ta sama cena biletu, plus opłata rezerwacyjna 2.00. */
public final class OnlineTicketPricing extends TicketPricing {
    @Override
    protected BigDecimal bookingFee() {
        return new BigDecimal("2.00");
    }
}

File: csharp/src/Training.Workshop/M4/S06InlineMethod/Step2/OnlineTicketPricing.cs
namespace Training.Workshop.M4.S06InlineMethod.Step2;

/// <summary>Sprzedaż internetowa: ta sama cena biletu, plus opłata rezerwacyjna 2.00.</summary>
public sealed class OnlineTicketPricing : TicketPricing
{
    protected override decimal BookingFee()
    {
        return 2.00m;
    }
}
